src/routes/(public)/auth/login/+page.svelte

```
@@ -144,7 +144,7 @@
 <div class="grid min-h-svh lg:grid-cols-2">
     <div class="flex flex-col gap-4 p-6 md:p-10">
         <div class="flex justify-center gap-2 md:justify-start">
-            <!-- svelte-ignore svelte/no-navigation-without-resolve -->
+            <!-- svelte-ignore a11y_no_static_element_interactions -->
```

ESLint

[error] svelte-ignore comment is used, but not warned

(svelte/no-unused-svelte-ignore)